dpnp/backend/kernels/dpnp_krnl_indexing.cpp

```diff
@@ -175,6 +175,25 @@ void dpnp_fill_diagonal_c(void* array1_in, void* val_in, size_t* shape, const si
     return;
 }
 
+template<typename _DataType, typename _MaskType>
+void dpnp_place_c(void* arr_in, void* mask_in, void* vals_in, const size_t arr_size, const size_t vals_size)
+{
+    _DataType* arr = reinterpret_cast<_DataType*>(arr_in);
```

Comment:
maybe it is better to reuse `copyto` kernel?

Reply:
`copyto` sounds a good option to cast input `mask` to required type if I get right.